### DISCUSSION TITLE
List of standard values for <osmand:icon> <osmand:background> tags?

### DISCUSSION THREAD

**Discussion by gbrown2036** [osmandapp/OsmAnd#19833] [Maps Related]
I looked through the documentation but don't see where it lists the valid values (or how to define the valid values and icons) for the `<osmand:icon>` and `<osmand:background>` tags.

This [example ](https://docs.osmand.net/docs/user/personal/favorites/#favorites-in-gpx-file)shows a value of `place_town` for `<osmand:icon>` but where do I find the rest of the valid values?

Is there a predefined list of values for these tags or instructions on how to define them?

Thanks!

**Comment by gbrown2036** [comment]
I tried editing a favorite to change the icon to a BEER icon but when I go to export it will only export as an OSF file which is binary so I can't decipher the tag by trial-and-error.

**Comment by mariush444** [comment]
List of icon that you can use in osmand's gpx
https://github.com/mariush444/gmapIcons2osmand/blob/main/icons-gmap-osmand.pdf

Background is: circle, square, octagon.
